zrp/prepare/geo_geocoder.py

```diff
@@ -242,7 +228,10 @@ def transform(self, input_data, geo, processed, replicate, save_table=True):
         print("   [Completed] Validating input geo data")        
 
         cols_to_drop = ['BLKGRPCE', 'COUNTYFP', 'FROMHN', 'TOHN', 'TRACTCE', 'ZCTA5CE', 'ZCTA5CE10', 'ZEST_FULLNAME', 
-                        'ZEST_ZIP', 'ZEST_KEY_LONG', 'replicate_flg', 'FROMHN_numeric', 'TOHN_numeric', 'house_numer_numeric']
+                        'ZEST_ZIP', 'ZEST_KEY_LONG', 'replicate_flg', 'FROMHN_LEFT', "FROMHN_RIGHT", 'TOHN_LEFT', 
```

Comment:
Is 'ZCTA5CE10' used?

Reply:
There were a lot of changes around ZCTA5CE10 in branch “Feat/zip asc source fix” so let’s hold on with this discussion until we merge that branch.